docs/server/commands.rst

```diff
@@ -431,13 +431,18 @@ no actual data will be loaded or deleted (the TM will be left unchanged):
 
 This command also allows to read translations from files and build the TM
 resources in the external TM server. In order to do so it is mandatory to
-provide the :option:`--tm` and :option:`--project` options, along with some
-filename argument to import translations from. The provided project name will
-be displayed on TM matches in the translation editor:
+provide the :option:`--tm` and :option:`--display-name` options, along with
+some filename argument to import translations from.
+
+The display name is a label used to group translations within a TM. A given TM
+can host translations for several display names. A use for the display name
+might be to specify the project from which the translation comes from. The
+display name will be shown on TM matches in the translation editor. Just
+specify it with :option:`--display-name`:
```

Comment:
The display name can be used to specify the name of the project from which the translations originate.

To specify a name use --display-name